inode/locker.go

```diff
@@ -0,0 +1,95 @@
+package inode
```

Comment:
Ok... so to follow the consistency of package Inode, I would have expected to see these APIs listed in inode/api.go. Further, rather than passing the volumeName as an argument, these should have been "methods" of the inode.volumeStruct, right? Sure, package fs needs to call them too I suppose... but in every case it does so, it has the inode VolumeHandle, right?

Reply:
Done